presto-docs/src/main/sphinx/functions/datetime.rst

```diff
@@ -374,4 +382,4 @@ Convenience Extraction Functions
 
 .. function:: yow(x) -> bigint
 
-    This is an alias for :func:`year_of_week`.
+    This is an alias for :func:`year_of_week`.
```

Comment:
Agreed, i thought if i introduced a blank space by mistake and tried removing it but can't find the difference.